modules/org.restlet.test/src/org/restlet/test/ext/apispark/conversion/swagger/v2_0/Swagger2TestCase.java

```diff
@@ -212,20 +213,20 @@ protected Definition createDefinition() {
         return definition;
     }
 
-    private void compareMapOfStrings(Map<String, String> savedMap,
-            Map<String, String> translatedMap) {
-        if (savedMap != null || translatedMap != null) {
-            assertNotNull(savedMap);
-            assertNotNull(translatedMap);
-            for (Entry<String, String> entry : savedMap.entrySet()) {
-                assertEquals(savedMap.get(entry.getKey()),
-                        translatedMap.get(entry.getKey()));
-            }
-            for (Entry<String, String> entry : translatedMap.entrySet()) {
-                assertEquals(savedMap.get(entry.getKey()),
-                        translatedMap.get(entry.getKey()));
-            }
+    private void compareMapOfStrings(Map<String, Object> savedMap,
+            Map<String, Object> translatedMap) {
+        if (savedMap == null && translatedMap == null) {
```

Comment:
It was functionally correct when the values were Strings but is not anymore. 
Anyway, it is used only to test Swagger's `Response#examples` which is not supported in RWADef yet. 
I suggest we delete the method.